timesketch/lib/analyzers/contrib/misp_analyzer.py

```diff
@@ -28,29 +28,29 @@ def __init__(self, index_name, sketch_id, timeline_id=None, **kwargs):
             timeline_id: The ID of the timeline.
         """
         super().__init__(index_name, sketch_id, timeline_id=timeline_id)
-        self.misp_url = kwargs.get("misp_url")
-        self.misp_api_key = kwargs.get("misp_api_key")
+        self.misp_url = current_app.config.get("MISP_URL")
+        self.misp_api_key = current_app.config.get("MISP_API_KEY")
         self.total_event_counter = 0
-        self.request_set = set()
         self.result_dict = dict()
+        self._query_string = kwargs.get("query")[0]
+        self._attr = kwargs.get("query")[1]
+        self._timesketch_attr = kwargs.get("query")[2]
 
     @staticmethod
     def get_kwargs():
         """Get kwargs for the analyzer.
 
         Returns:
-            Info to connect to MISP.
+            List of attributes to search for in Timesketch and query in MISP
         """
 
-        misp_url = current_app.config.get("MISP_URL")
-        misp_api_key = current_app.config.get("MISP_API_KEY")
-
-        if not misp_api_key or not misp_url:
-            logger.error("MISP conf not found")
-            return []
-
-        matcher_kwargs = [{"misp_url": misp_url, "misp_api_key": misp_api_key}]
-        return matcher_kwargs
+        to_query = [
+            {"query": ["md5_hash:*", "md5", "md5_hash"]},
+            {"query": ["sha1_hash:*", "sha1", "sha1_hash"]},
+            {"query": ["sha256_hash:*", "sha256", "sha256_hash"]},
+            {"query": ["filename:*", "filename", "filename"]},
+        ]
```

Comment:
Optional suggestion to make it easier to read the code:

```suggestion
        to_query = [
            {"query_string": "md5_hash:*",  "attr": "md5", "timesketch_attr": "md5_hash"},
            {"query_string": "sha1_hash:*",  "attr": "sha1", "timesketch_attr": "sha1_hash"},
            {"query_string": "sha256_hash:*", "attr": "sha256", "timesketch_attr": "sha256_hash"},
            {"query_string": "filename:*", "attr": "filename", "timesketch_attr": "filename"},
        ]
```